packages/workbox-build/src/lib/get-file-details.js

```diff
@@ -21,12 +21,22 @@ const errors = require('./errors');
 const getFileSize = require('./get-file-size');
 const getFileHash = require('./get-file-hash');
 
-module.exports = ({globDirectory, globIgnores, globPattern}) => {
+module.exports = (globOptions) => {
+  const {
+    globDirectory,
+    globIgnores,
+    globPattern,
+    globFollow,
+    globStrict,
+  } = globOptions;
   let globbedFiles;
   try {
     globbedFiles = glob.sync(globPattern, {
       cwd: globDirectory,
       ignore: globIgnores,
+      follow: typeof globFollow !== 'undefined'? globFollow : true,
```

Comment:
@jeffposnick you've been doing some work on the defaults, is this the best approach or should this be moved to a defaults value.